api/src/pcapi/utils/secrets.py

```diff
@@ -11,15 +11,15 @@
 
 def get(key: str, default: str | None = None) -> str:
     """
-    Request a secret by it's key.
+    Request a secret by its key.
 
-    This function add the key to the SECRET_KEYS list so that the secret will be requested and injected as an environnement variable.
-    It's value is then extracted from environnement variables.
+    This function add the key to the SECRET_KEYS list so that the secret will be requested and injected as an environment variable.
```

Comment:
```suggestion
    This function adds the key to the SECRET_KEYS list so that the secret will be requested and injected as an environment variable.
```